custom-connectors/Xbridger/apiDefinition.swagger.json

```diff
@@ -0,0 +1,498 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1",
+    "title": "XbridgerDocumentManager",
```

Comment:
Could spaces be added in the title? So something to the effect of `Xbridger Document Manager`?

Reply:
@dougelliott, The swagger.json file has been updated and solution moved to certified-connector folder.